### DISCUSSION TITLE
Visualize the segmentation results in Python?

### DISCUSSION THREAD

**Discussion by ywu202300** [BodenmillerGroup/steinbock#211] [Q&A]
Hi,

In the steinbock pipeline, is there a way to visualize the segmentation results from steinbock with napari or other Python tools

**Comment by Milad4849** [comment]
Hi @ywu202300 

Please have a look at the [visualization section](https://bodenmillergroup.github.io/steinbock/latest/cli/visualization/) of the documentation.

**Comment by jwindhager** [comment]
https://github.com/BIIFSweden/steinbock-webinar/blob/main/napari.ipynb

https://github.com/BodenmillerGroup/ImagingWorkshop2023/blob/main/Session1_image_visualization/napari.ipynb
